### DISCUSSION TITLE
Adding primary color using `sx` prop

### DISCUSSION THREAD

**Discussion by thevipinmishra** [mantinedev/mantine#2318] [Q&A] [answered]
I'm trying to add primary color on my <Title/> component using the  `sx` prop.

So, I'm doing something like this:

`  <Title sx={(theme) => ({ color: theme.primaryColor })} weight={700}>120</Title>`

I'm expecting the color the be my primaryColor value with shade that I've provided on <MantineProvider />

Here's my Provider:

` <MantineProvider
      withCSSVariables
      withGlobalStyles
      withNormalizeCSS
      theme={{
        fontFamily: "Inter",
        primaryColor: "green",
        primaryShade: 6,
        headings: {
          fontFamily: "Inter",
        },
      }}`
      
      
      Can I achieve this? or it won't work and I'll have to hard code the theme color?

**Comment by rtivital** [comment, marked as answer]
If you are using latest version you can use `theme.fn.primaryColor()`, for older versions use `theme.colors[theme.primaryColor][theme.fn.primaryShade()]`

**Reply by reorx** [reply]
Thanks! I just know you won't forget to provide this function, but I think it's better to mention it in the "Colors > Primary color" section, people are expecting to see how to get the actual color code here :)
